odoo/addons/base/models/ir_model.py

```diff
@@ -623,6 +623,26 @@ def _onchange_related(self):
             self.relation = field.relation
             self.readonly = True
 
+    def _validate_relation_field(self):
+        model_name = self.relation
+        model_id = self.env['ir.model']._get_id(model_name)
+        if not model_id:
+            raise UserError(_("Unknown model name '%s' in relation model") % (model_name))
```

Comment:
See the API of function `_`, it follows the "logger" API:
```suggestion
            raise UserError(_("Unknown model name '%s' in relation model", model_name))
```